cif_core.dic

```diff
@@ -27546,17 +27667,27 @@ save_
        Changed the purpose of the _diffrn_radiation_wavelength.id data item
        from 'Encode' to 'Key'.
 
+       Reverted _atom_type_scat.versus_stol_list to a text-type to regain
```

Comment:
Part of the changelog message is assigned to version 3.2.0 and part to 3.3.0. These should be moved to a single version.